### DISCUSSION TITLE
Why Karavan has a special section for GITEA

### DISCUSSION THREAD

**Discussion by manoger** [apache/camel-karavan#1286] [Q&A] [answered]
### Ask a question

Firstly, Hello 👋
Hope you're doing well.

In this [how-to run in docker](https://github.com/apache/camel-karavan/blob/main/docs/WEB_DOCKER.md), there's a specific section on "How to run Demo Karavan on Docker with Gitea", why?

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by mgubaidullin** [comment, marked as answer]
Karavan uses git repository to store project files. So to setup Karavan you need to have an empty git repository.

"Demo Karavan on Docker with Gitea" is a good starting point, because it required only one prerequisite: Docker.
It comer with container image with Gitea and pre-created repository.
